pycognito/aws_srp.py

```diff
@@ -243,7 +243,7 @@ def process_challenge(self, challenge_parameters):
             )
         return response
 
-    def authenticate_user(self, client=None):
+    def authenticate_user(self, client=None, metadata=None):
```

Comment:
please keep the naming of client_metadata to avoid confusion with user attributes that we use as metadata

Reply:
Fixed!